statsmodels/regression/tests/test_lme.py

```diff
@@ -417,8 +440,8 @@ def test_dietox(self):
         assert_allclose(result.cov_re, 39.82097, rtol=1e-5)
 
         # logLik(rm)
```

Comment:
`logLik` is almost certainly out of date, doesn't serve as a very useful comment.  Not sure what its replacement would be.

Reply:
These comments are the R code required to extract the relevant values for comparison.